### ISSUE TITLE
`/plot list` Usage of MiniMessage by list item

### ISSUE THREAD

**Issue by RedstoneFuture** [IntellectualSites/PlotSquared#4289]
### Server Implementation

Paper

### Server Version

1.19.4

### Describe the bug

`  "info.plot_user_list": "<click:suggest_command:'<user>'><hover:show_text:'<gray>Copy user name'><white><user><gray></hover></click>",`

![grafik](https://github.com/IntellectualSites/PlotSquared/assets/4140635/0ad3e8e5-af6d-4e75-a44d-147a4122d680)

![grafik](https://github.com/IntellectualSites/PlotSquared/assets/4140635/9866b1b4-830e-4378-96ea-17de8f6d02fd)

### To Reproduce

`messages_en.json` ->

```
`  "info.plot_user_list": "<click:suggest_command:'<user>'><hover:show_text:'<gray>Copy user name'><white><user><gray></hover></click>",`
```

### Expected behaviour

![grafik](https://github.com/IntellectualSites/PlotSquared/assets/4140635/ab13d8da-562d-4a7b-b89c-c6bb143b54a0)

### Plot Debugpaste

XXX

### PlotSquared Version

PlotSquared 7.2.0

### Checklist

- [X] I have included a Plot debugpaste.
- [X] I am using the newest build from https://www.spigotmc.org/resources/77506/ and the issue still persists.

**Comment by PierreSchwang** [comment]
Relating issue in adventure repository: https://github.com/KyoriPowered/adventure/issues/964

It's due to the fact, that we insert styled components and such tag arguments require plain processed string values. Could be a feature request for supply one tag (e.g. `user_raw`) as well I guess,
What does work locally is using `Tag.preProcessParsed` - that will be inserted as expected.

I did not push any changes as I'm not sure if there are any objections due to adding the plain-text-serializer and a new argument (where the naming might be better)

**Comment by RedstoneFuture** [comment]
`user_raw` and `uuid_raw`.
I would like to have the UUID in the suggest-command later on.

**Comment by PierreSchwang** [comment]
> `user_raw` and `uuid_raw`.
> 
> I would like to have the UUID in the suggest-command later on.

You might run into issues later on with that. The <user> can be a comma separated list of usernames as well which potentially interferes with your plan.

**Comment by RedstoneFuture** [comment]
But `info.plot_user_list` is for all usernames separately, as he defines the list items. It cannot be "comma separated list of usernames" here.

**Comment by PierreSchwang** [comment]
> But `info.plot_user_list` is for all usernames separately, as he defines the list items. It cannot be "comma separated list of usernames" here.

Oh yeah, you are right. Sorry, my bad. I guess just adding the two tags should suffice in the beginning - MM does not really provide any kind of conditional tags with ternary operations etc, so we can't really differentiate between player-owned and server/everyone-owned plots.

**Comment by RedstoneFuture** [comment]
> Edit: The user can be the translation for "Everyone" and "Server" as well, where no handling in such cases is clear to me. I mean, they do have internal UUIDs, but I doubt those should be recommend a command?

Yes, that's right. Everyone (`*`) is saved as UUID = `00000001-0001-0003-0003-000000000007`.
But in this case, a suggest-command of this UUID is acceptable, I think.
